# Generated by Django 3.2.6 on 2023-04-14 08:57

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('apigw', '0019_apiservicelog_error'),
    ]

    operations = [
        migrations.AddField(
            model_name='apiservice',
            name='oauth2_api_ocp_apim_subscription_key',
            field=models.CharField(blank=True, default='', max_length=256, null=True),
        ),
        migrations.AddField(
            model_name='apiservice',
            name='oauth2_api_url',
            field=models.CharField(blank=True, default='', max_length=256, null=True),
        ),
        migrations.AddField(
            model_name='apiservice',
            name='oauth2_client_id',
            field=models.CharField(blank=True, default='', max_length=256, null=True),
        ),
        migrations.AddField(
            model_name='apiservice',
            name='oauth2_secret',
            field=models.CharField(blank=True, default='', max_length=256, null=True),
        ),
        migrations.AddField(
            model_name='apiservice',
            name='oauth2_url',
            field=models.CharField(blank=True, default='', max_length=256, null=True),
        ),
    ]
